src/server/scripts/EasternKingdoms/SunkenTemple/sunken_temple.h

```diff
@@ -23,9 +23,64 @@
 #define STScriptName "instance_sunken_temple"
 #define DataHeader "ST"
 
-enum STEvents
+uint32 const EncounterCount = 1;
+
+enum STDataTypes
+{
+    DATA_AVATAR_OF_HAKKAR = 0,
+
+    DATA_ETERNAL_FLAME    = 9,
+    EVENT_STATE           = 10
+};
+
+enum STGameObjectIds
+{
+    GO_ATALAI_STATUE1 = 148830,
+    GO_ATALAI_STATUE2 = 148831,
+    GO_ATALAI_STATUE3 = 148832,
+    GO_ATALAI_STATUE4 = 148833,
+    GO_ATALAI_STATUE5 = 148834,
+    GO_ATALAI_STATUE6 = 148835,
+    GO_ATALAI_LIGHT1 = 148883,
+    GO_ATALAI_LIGHT2 = 148937,
+
+    GO_ETERNAL_FLAME_1 = 148418,
+    GO_ETERNAL_FLAME_2 = 148419,
+    GO_ETERNAL_FLAME_3 = 148420,
+    GO_ETERNAL_FLAME_4 = 148421,
+    GO_EVIL_GOD_SUMMONING_CIRCLE = 148998,
+    GO_AVATAR_OF_HAKKAR_DOOR     = 149432,
```

Comment:
Even though it can look nice to group the enums in consecutive groups, consistency is preferred.
Please align all values in this enum group (`STGameObjectIds`).